plugins/importexport/medra/MedraInfoSender.inc.php

```diff
@@ -0,0 +1,157 @@
+
+	/**
+	 * Constructor.
+	 * @param $argv array task arguments
+	 */
+	function MedraInfoSender($args) {
+		PluginRegistry::loadCategory('importexport');
+		$plugin =& PluginRegistry::getPlugin('importexport', 'MedraExportPlugin'); /* @var $plugin MedraExportPlugin */
+		$this->_plugin =& $plugin;
+
+		if (is_a($plugin, 'MedraExportPlugin')) {
+			$plugin->addLocaleData();
+		}
+
+		parent::ScheduledTask($args);
+	}
+
+	/**
+	 * @see ScheduledTask::getName()
+	 */
+	function getName() {
+		return __('plugins.importexport.medra.senderTask.name');
+	}
+
+	/**
+	 * @see ScheduledTask::executeActions()
+	 */
+	function executeActions() {
+		if (!$this->_plugin) return false;
+
+		$plugin = $this->_plugin;
+		$journals = $this->_getJournals();
+		$request =& Application::getRequest();
+		foreach ($journals as $journal) {
+			$unregisteredIssues = $plugin->_getUnregisteredIssues($journal);
+			$unregisteredArticles = $plugin->_getUnregisteredArticles($journal);
+			$unregisteredGalleys = $plugin->_getUnregisteredGalleys($journal);
+			$errors = array();
+
+			$unregisteredIssueIds = array();
+			foreach ($unregisteredIssues as $issue) {
+				if ($plugin->canBeExported($issue, $errors)) {
+					$unregisteredIssueIds[] = $issue->getId();
+				}
+			}
+			$unregisteredArticlesIds = array();
+			foreach ($unregisteredArticles as $articleData) {
+				$article = $articleData['article'];
+				if (is_a($article, 'PublishedArticle') && $plugin->canBeExported($article, $errors)) {
+					$unregisteredArticlesIds[] = $article->getId();
+				}
+			}
+			$unregisteredGalleyIds = array();
+			foreach ($unregisteredGalleys as $galleyData) {
+				$galley = $galleyData['galley'];
+				if ($plugin->canBeExported($galley, $errors)) {
+					$unregisteredGalleyIds[] = $galley->getId();
+				}
+			}
+
+			// If there are unregistered DOIs and we want automatic deposits
+			$exportSpec = array();
+			$register = false;
+			if (count($unregisteredIssueIds)) {
+				$exportSpec[DOI_EXPORT_ISSUES] = $unregisteredIssueIds;
+				$register = true;
+			}
+			if (count($unregisteredArticlesIds)) {
+				$exportSpec[DOI_EXPORT_ARTICLES] = $unregisteredArticlesIds;
+				$register = true;
+			}
+			if (count($unregisteredGalleyIds)) {
+				$exportSpec[DOI_EXPORT_GALLEYS] = $unregisteredGalleyIds;
+				$register = true;
+			}
+
+			if ($register) {
+				$result = $plugin->registerObjects($request, $exportSpec, $journal);
+				if ($result !== true) {
+					if (is_array($result)) {
+						foreach($result as $error) {
+							assert(is_array($error) && count($error) >= 1);
+							$this->addExecutionLogEntry(
+								__($error[0], array('param' => (isset($error[1]) ? $error[1] : null))),
+								SCHEDULED_TASK_MESSAGE_TYPE_WARNING
+							);
+						}
+					} else {
+						$this->addExecutionLogEntry(
+							__('plugins.importexport.common.register.error.mdsError', array('param' => ' - ')),
+							SCHEDULED_TASK_MESSAGE_TYPE_WARNING
+						);
+					}
+				}
+ 			}
+  		}
+			return true;
```

Comment:
Looks like incorrect indentation? (Sorry I didn't spot this over on `ojs-dev-2_4`)